esp/src/eclwatch/templates/GetUserName.html

```diff
@@ -0,0 +1,126 @@
+
+            .login input {
+                margin-bottom: 20px;
+                width: 300px;
+                padding: 8px;
+                border: 1px solid #bfbfbf;
+            }
+
+            .login form {
+                margin: auto;
+                width: 300px;
+            }
+
+            img {
+                display: block;
+                margin: auto;
+            }
+
+            p {
+                text-align: center
+            }
+
+            #button {
+                background-color: #1A9BD7;
+                border: none;
+                color: white;
+                padding: 15px 32px;
+                text-align: center;
+                text-decoration: none;
+                display: inline-block;
+                font-size: 16px;
+                border-radius: 4px;
+                cursor: pointer;
+                -webkit-transition-duration: 0.4s;
+                /* Safari */
+                transition-duration: 0.4s;
+                margin: auto;
+                display: block;
+            }
+
+            .button:hover {
+                background-color: #13b1f9;
+                color: white;
+            }
+
+            .visible {
+                display: none;
+            }
+        </style>
+        <script>
+            dojoConfig = {
+                packages: [
+                    { name: "eclwatch", location: "/esp/files/eclwatch" }
+                ]
+            };
+        </script>
+        <script src="../../dojo/dojo.js"></script>
+        <script type="text/javascript">
+            require(["dojo", "eclwatch/nls/hpcc", "dojo/i18n", "dojo/i18n!hpcc/nls/hpcc", "dojo/query", "dojo/cookie"], function (dojo, hpcc, i18n, nlsHPCC, query, cookie) {
+                if (dojo.cookie.isSupported()){
+                    dojo.replaceClass("container", "container", "container  visible");
+                    query(".loginStr").forEach(function (node, index, arr) {
+                        switch (index) {
+                            case 0:
+                            case 1:
+                                node.innerHTML = nlsHPCC.UserLogin
+                                break;
+                            case 2:
+                                node.placeholder = nlsHPCC.Username
+                                break;
+                            case 3:
+                                node.innerHTML = nlsHPCC.Login
+                                break;
+                        }
+                    });
+                } else {
+                    alert(nlsHPCC.PleaseEnableCookies);
+                }
+            });
+        </script>
+    </head>
+    <body>
+        <div id="container" class="container visible">
+            <div class="formContainer">
+                <img src="../img/Loginlogo.png" alt="HPCC Systems">
+                <p class="loginStr"></p>
+                <div class="login">
+                    <form method="POST" action="/esp/login">
```

Comment:
@wangkx Do we still need to call /esp/login when just sending username?

Reply:
Do not have to be 'login'. You may change to different wording.

Reply:
Does it have to post to anything? I do not think it needs to call an `action=""` at all?

Reply:
I think we should identify this action (not using now but may use in the future): /esp/setusername.

Reply:
We cannot point to an empty action that does not exist. It will simply redirect the page to itself.

Reply:
If ESP has logic to know if user is not LDAP enabled and will handle accordingly, this looks fine.